### DISCUSSION TITLE
How to show message box in Open Mct

### DISCUSSION THREAD

**Discussion by Vaibhav2121** [nasa/openmct#4742] [Community Q&A]
If some old data is not available . I want to show message in Open Mct through message box. How to do it.

**Comment by jvigliotta** [comment]
@Vaibhav2121 Hey! Good question, we have a couple different ways to go about this. We have a notification API you could use: https://github.com/nasa/openmct/blob/master/src/api/notifications/NotificationAPI.js and we also have an overlay API that could work:  https://github.com/nasa/openmct/blob/master/src/api/overlays/OverlayAPI.js

Notifications appear up in the top bar. Overlays appear as a window/box.

Usage Examples:
`openmct.notifications.info('message text', opts)`
`openmct.notifications.alert('message text', opts)`
`openmct.notifications.error('message text', opts)`

and

`openmct.overlays.overlay(opts)`
`openmct.overlays.dialog(opts)`

You may want to look at using the notification API as it's more informative and less blocking, but you may find the dialog method of the overalys api useful as well. Let us know what you end up with!

**Reply by Vaibhav2121** [reply]
how can we access openmct object in realtime-server.js and history-server.js
